frontend/stylesheets/goldenlayout_overwrites.less

```diff
@@ -86,3 +86,12 @@
     border-bottom: 1px solid #1890ff;
   }
 }
+
+.hidden-gl-item {
```

Comment:
Can you add a comment, that this is used to hide panes when a pane is maximized?